docs/news-widget/web-components/sesamy-content-container.md

```diff
@@ -98,12 +98,18 @@ When using the `lock-mode=signedUrl` lock mode the content is fetched from the s
 >
 ```
 
-### Unlock Message
+### Receipt Link
 
 This message is shown below the article once it is unlocked, it contains a link to the confirmation screen of the checkout where te item was purchased.
-It can be hidden by including the attribute `hide-unlock-message` in the sesamy-content-container tag (`<sesamy-content-container hide-unlock-message>`).
+It can be configured by setting the attribute `receipt-link` in the sesamy-content-container tag (`<sesamy-content-container receipt-link="none">`).
 
-The styling of the unlocke message can be configured with the following css variables:
+The following options are available fo the `receipt-link` attribute:
+
+- Text (default) (`receipt-link="text"`): Displays a `powered by Sesamy` message
+- Embed (`receipt-link="embed"`): Displays a receipt in a embedded window
```

Comment:
`aN embedded window` (word starts with a vowel) :upside_down_face: 

Apologies for being pedantic but the docs are customer facing!

Reply:
I actually incorrectly thought that `a` or `an` was decided by the noun and not the adjective :) In other languages you typically have a genus which won't change depending on the adjective. And in Swedish we have the same behaviour with `en` or `ett` but it's also defined by the noun. Interesting!